capstone-ui/src/components/SongHeaderView/SongHeaderView.jsx

```diff
@@ -20,9 +20,13 @@ export default function SongHeaderView({ song }) {
   return (
     <div className="songheader-component">
       <div className="play-song-wrapper">
-        {song.album && song.album.images && (
-          <img src={song.album.images[0].url} alt="song-image"></img>
-        )}
+        <iframe
+          className="songheader-playing-picture"
+          src={`https://open.spotify.com/embed/track/${song.id}`}
+          width="100%"
+          height="380"
+          frameBorder="0"
+        ></iframe>
```

Comment:
```suggestion
        </iframe
          className="songheader-playing-picture"
          src={`https://open.spotify.com/embed/track/${song.id}`}
          width="100%"
          height="380"
          frameBorder="0"
        >
```

Reply:
I will go through all my components after merging them all to format them like this